ideascube/db_router.py

```diff
@@ -1,30 +1,44 @@
+
+_databaseToUse = {
+    ('search', 'search') : 'transient'
```

Comment:
I disagree with you here :)

I don't see any reason to have two nested dictionaries.
And it forces us to have two lookup instead of one.